Mesh_3/include/CGAL/Polyhedral_mesh_domain_with_features_3.h

```diff
@@ -406,6 +409,14 @@ detect_features(FT angle_in_degree, std::vector<Polyhedron>& poly)
   if (borders_detected_)
     return;//prevent from not-terminating
 
+  typedef Featured_edges_copy_graph G_copy;
+  G_copy g_copy;
+  typedef typename boost::graph_traits<G_copy>::vertex_descriptor vertex_descriptor;
+  typedef std::map<typename Polyhedron::Point,
+                   vertex_descriptor> P2vmap;
+  // TODO: replace this map by and unordered_map
```

Comment:
Yes, exactly: polyhedron vertices (from different polyhedra or not) with the same coordinates are assigned to the same vertex of the graph.

... I could transform our CGAL `Point_3` to `CGAL::cpp11::array<double, 3>` and use the standard hash functions from Boost.